src/FungibleTokenContract.ts

```diff
@@ -1406,3 +1550,25 @@ function toForest(
   );
   return AccountUpdateForest.fromReverse(trees);
 }
+
+// =============================================================================
+// EXPORTS
+// =============================================================================
+
+export {
```

Comment:
I'm curios why you decided to move all the exports to the bottom of the file? We export at the top of the files in other repos like o1js so it is immediately apparent what is being exported for  https://github.com/o1-labs/o1js/blob/main/src/lib/provable/core/field-constructor.ts#L10-L20 .
Both work though

Reply:
Makes sense, thanks.